### ISSUE TITLE
Feature/Example Request: Checkbox Select All

### ISSUE THREAD

**Issue by Nick-Zuchlewski** [AhmedLSayed9/dropdown_button2#167]
I am building a dropdown using checkboxes from the sample on pub dev. I would like to be able add a All/None checkbox on top to select all checkboxes. Can you please provide a sample or create a feature?

**Comment by guchengxi1994** [comment]
@AhmedLSayed9 
I used `clear` and `addAll`, but cannot repaint the DropdownMenuItem. Here is my code
![image](https://github.com/AhmedLSayed9/dropdown_button2/assets/33513462/71903a7d-e0d2-4e1f-928b-b67d6e4e5a8d)

**Comment by AhmedLSayed9** [comment]
@guchengxi1994 You're missing `setState`. Check [Example 4](https://github.com/AhmedLSayed9/dropdown_button2#4-dropdownbutton2-as-multiselect-dropdown-with-checkboxes)

**Comment by guchengxi1994** [comment]
@AhmedLSayed9  thanks for response. I used `setState` in the Inkwell , `addAll` not work but add/remove a single item works as the Example4 shows. what have I missed？
![image](https://github.com/AhmedLSayed9/dropdown_button2/assets/33513462/d91f77c5-a12c-41a0-91df-a2b2b2800d14)

I checked the `selectedItems` every time in the `build` function, after `addAll` called, `selectedItems` changes but `DropdownMenuItem` not repaint.
